From 9e83ac0fb966eadcb8c4401b70e3a58a7dc87fa5 Mon Sep 17 00:00:00 2001
From: Woody Zhang <woodylab@foxmail.com>
Date: Mon, 1 May 2023 15:24:06 +0800
Subject: [PATCH] Makefile:add modules_install make target

modules_install is a standand make target for out-of-tree kernel
modules.

Upstream-Status: Pending
Signed-off-by: Woody Zhang <woodylab@foxmail.com>
---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index ddd4561..468e169 100644
--- a/Makefile
+++ b/Makefile
@@ -1802,6 +1802,9 @@ all: modules
 modules:
 	$(MAKE) ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE) -C $(KSRC) M=$(shell pwd)  modules
 
+modules_install:
+	$(MAKE) ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE) -C $(KSRC) M=$(shell pwd)  modules_install
+
 strip:
 	$(CROSS_COMPILE)strip $(MODULE_NAME).ko --strip-unneeded
 
